src/apic-extension/azext_apic_extension/aaz/latest/apic/metadata/_export.py

```diff
@@ -25,6 +25,10 @@ class Export(AAZCommand):
 
     :example: Export Metadata Schema assigned to environment
         az apic metadata export -g api-center-test -n contosoeuap --assignments environment --file-name filename.json
+
+    :example: Export Custom Metadata Schema Only
+        az apic metadata export -g contoso-resources -n contoso --assignments api --file-name filename.json --custom-metadata-only
+        az apic metadata export -g contoso-resources -n contoso --assignments api --file-name filename.json --custom-metadata-only true
```

Comment:
Please remove this example, as using `--custom-metadata-only` without any value is more preferred

Reply:
ok